src/borg/repository.py

```diff
@@ -749,7 +749,7 @@ def check_free_space(self):
                     try:
                         compact_working_space += self.io.segment_size(segment) - free
                     except FileNotFoundError:
-                        # looks like self.compact is referring to a non-existent segment file, ignore it.
+                        # looks like self.compact is referring to a nonexistent segment file, ignore it.
```

Comment:
https://dict.leo.org/german-english/non-existent seems to be BE vs AE.

I guess our docs are a wild mix between BE and AE...